assets/components/paymentButtons/payPalExpressButton/payPalExpressButton.jsx

```diff
@@ -59,12 +62,25 @@ function Button(props: PropTypes) {
 
   const PayPalButton = window.paypal.Button.driver('react', { React, ReactDOM });
 
+  if (props.disable) {
+    return (
+      <button
+        id="qa-pay-with-direct-debit"
+        className={classNameWithModifiers('component-paypal-fake-pop-up-button', ['disable'])}
```

Comment:
Are we using the disable modifier here? I don't find in the css a place where is the definition of that class with that modifier.